Description: Disable wget dependency as we do not wget dependencies at buildtime.
Author: Scarlett Moore <sgclark@kde.org>
Forwarded: not-needed
Last-Update: 2019-01-11
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: libappimage-0.1.6/cmake/tools.cmake
===================================================================
--- libappimage-0.1.6.orig/cmake/tools.cmake	2019-01-06 11:53:44.612905996 +0000
+++ libappimage-0.1.6/cmake/tools.cmake	2019-01-11 13:29:30.515685126 +0000
@@ -11,7 +11,7 @@
     if(NOT ARG_NAME)
         message(FATAL_ERROR "NAME argument required for check_program")
     endif()
-
+
     if(TOOLS_PREFIX)
         set(prefix ${TOOLS_PREFIX})
     endif()
@@ -50,7 +50,7 @@
 check_program(NAME libtoolize)
 check_program(NAME patch)
 check_program(NAME sed)
-check_program(NAME wget)
+#check_program(NAME wget)
 check_program(NAME xxd)
 check_program(NAME desktop-file-validate)
 check_program(NAME objcopy FORCE_PREFIX)
